static/src/javascripts/projects/common/modules/onward/history.js

```diff
@@ -462,6 +464,45 @@ const showInMegaNavEnable = (bool: boolean): void => {
     saveSummary(summary);
 };
 
+const incrementDailyArticleCount = (pageConfig: Object): void => {
+    if (!pageConfig.isFront) {
+        const dailyCount = local.get(storageKeyDailyArticleCount) || [];
+
+        if (dailyCount[0] && dailyCount[0].day && dailyCount[0].day === today) {
+            dailyCount[0].count += 1;
+        } else {
+            // New day
+            dailyCount.unshift({ day: today, count: 1 });
+
+            // Remove any days older than 30
+            const cutOff = today - 30;
+            const firstOldDayIndex = dailyCount.findIndex(
+                c => c.day && c.day < cutOff
+            );
+            if (firstOldDayIndex > 0) {
+                dailyCount.splice(firstOldDayIndex);
+            }
+        }
+
+        local.set(storageKeyDailyArticleCount, dailyCount);
+    }
+};
+
+const getArticleViewCount = (days: number): number => {
+    const dailyCount = local.get(storageKeyDailyArticleCount) || [];
+    const cutOff = today - days;
+
+    const firstOldDayIndex = dailyCount.findIndex(
+        c => c.day && c.day <= cutOff
```

Comment:
I find the combination of the arrow function and the `less than or equals` on the same line slightly confusing to read. I read the arrow function as 'greater than or equals' at first. Not sure if this is linter-friendly, but would be great if you could move the `c =>` to the line above.

Reply:
good point - but unfortunately the linter does not allow it :(

Reply:
👍